src/lexer/update.ts

```diff
@@ -0,0 +1,46 @@
+import { Keyword } from "./tokens";
+import { ILexer } from "./interface";
+import { Query, Line } from "../reader/reader";
+import { cleanUnquotedIdentifier } from "./lexer";
+
+class Update implements ILexer {
+    private lastToken: string|null = null;
+
+    public tokenise(query: Query): Query {
+        this.lastToken = "";
+        query.lines.forEach(this.findUpdateStatementInLine);
+        return query;
+    }
+
+    private findUpdateStatementInLine(line: Line): void {
+        line.content.split(" ").forEach(word => {
+            const item = word.toLowerCase();
+
+            if (Object.values(Keyword).includes(item)) {
+                /**
+                 * Why are we checking for all keywords here?
```

Comment:
You tell me, what are you trying to do?

Reply:
I've pretty much copied the logic from the other lexer files. They all check for all keywords :/